v3/plugins/eclipse/gwt/6.19.0/meta.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: v2
```

Comment:
What it apiVersion here? Should be it the same as root folder name? For this file it is `v3`

Reply:
Now I see that it should be `v2` here. More see https://github.com/eclipse/che-plugin-registry/pull/135#issuecomment-488632994